### ISSUE TITLE
aggregate_select() removed in Django 1.10

### ISSUE THREAD

**Issue by OrangeDog23** [lionheart/django-pyodbc#138]
aggregate_select() was replaced by annotation_select() in Django 1.8.
Need fix it in compiler.SQLCompiler._fix_aggregates.
In my case(MSSQL 2008) simple replace aggregate_select to annotation_select works well.

**Comment by benmehlman** [comment]
Regarding the above pull request:

In order to get my django application working I've patched my fork of django-pyodbc as suggested by OrangeDog23 and the solution seems to work.  My own knowledge of Django's database internals is not great enough to assure anyone that @OrangeDog23 's solution is a good one, I can only say that I implemented his suggestion with some attention to forward and backward compatibility and that it does (so far) seem to work for me.

Thanks!
Ben Mehlman
